Detectors/PHOS/reconstruction/src/Cluster.cxx

```diff
@@ -340,7 +340,7 @@ void Cluster::EvalPrimaries(const std::vector<Digit>* digits)
         }
       }
       if (!found) {
-        mLabelsEProp.push_back(lab);
+        mLabels.push_back(lab);
```

Comment:
@peressounko : Could you please check this? I think this was most likely a typo in the original version, but was not caught to to implicit conversion from a label to a double.